modules/planning/tasks/deciders/path_bounds_decider.cc

```diff
@@ -245,9 +248,10 @@ std::string PathBoundsDecider::GenerateFallbackPathBound(
   // PathBoundsDebugString(*path_bound);
 
   // 2. Decide a rough boundary based on lane info and ADC's position
-  if (!GetBoundaryFromLanesAndADC(reference_line_info.reference_line(),
+  std::string dummy;
```

Comment:
Do not name it dummy, make the name more self-explaining, this string is for "borrow_lane_type" right?

Reply:
Thx. I updated it to be dummy_borrow_lane_type. It is for "borrow lane type" but it is a dummy variable, too.